src/mmw/js/src/compare/controllers.js

```diff
@@ -16,28 +18,87 @@ var CompareController = {
             App.currProject
                 .fetch()
                 .done(function() {
+                    setupProjectCopy();
```

Comment:
> Consider fetching the project from the server when returning from the compare view to repopulate the page with the saved version of the project.

We can do that for saved projects, but not unsaved.

> Alternatively, consider doing the project "copying" a bit more dynamically and on an as-needed basis. For example, once a precipitation slider is changed for a scenario, have that scenario store it's current attributes in a temporary attribute. When the page is moved back to model, then apply the saved attributes to the changed scenario. I haven't used it much but model.previousAttributes and/or model.previous() may be of some use here.

I was thinking of doing that, but I think it makes the code more complicated. I'll think it about some more though.